packages/p2p-v2/src/constants/orders.ts

```diff
@@ -0,0 +1,13 @@
+export const ORDERS_STATUS = Object.freeze({
```

Comment:
[Suggestion]: better to use as const than Object.freeze for various reasons.

One reason is that object.freeze is a runtime thing whereas as const is purely in compile time.

You can have a look at this video for more details :) https://www.youtube.com/watch?v=jjMbPt_H3RQ